src/index.ts

```diff
@@ -41,6 +42,7 @@ export class TinyColor {
   gradientType?: string;
   /** rounded alpha */
   roundA!: number;
+  random = random;
```

Comment:
I was thinking this would be a `static`, so you could do `TinyColor.random()` instead of `new TinyColor.random()`. This is basically to support the v1 API (as in https://github.com/bgrins/TinyColor#random)

Reply:
Looking at TS docs, I think this would be done as:

```
static random = random;
```